src/ssl.c

```diff
@@ -15573,13 +15590,91 @@ int wolfSSL_ASN1_INTEGER_cmp(const WOLFSSL_ASN1_INTEGER* a,
     return 0;
 }
 
-
 long wolfSSL_ASN1_INTEGER_get(const WOLFSSL_ASN1_INTEGER* i)
 {
-    (void)i;
-    return 0;
+  int e;
+  signed_word64 r;
+
+  if (i == NULL){
+      WOLFSSL_MSG("input parameter is NULL");
+      return 0;
+  }
+
+  e = wolfSSL_ASN1_INTEGER_get_int64(&r, i);
+
+  if (e == 0){
+      return -1;
+  }
+
+  if (r > LONG_MAX || r < LONG_MIN){
+      WOLFSSL_MSG("result is larger than LONG_MAX or smaller than LONG_MIN");
+      return -1;
+  }
+
+  return (long)r;
 }
 
+int wolfSSL_ASN1_INTEGER_get_int64(signed_word64 *p, const WOLFSSL_ASN1_INTEGER *i)
```

Comment:
Since this function uses <stdint.h> functionality, it should be protected with "#ifndef NO_STDINT".